build_debian.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+echo "================================================"
+echo " Attention - Building snarkOS from source code."
+echo " This will request root permissions with sudo."
+echo "================================================"
+
+# Install Debian dependencies
+
+sudo apt-get update
+sudo apt-get install -y \
+	build-essential \
+	curl \
+	gcc \
+	libssl-dev \
+	make \
+	pkg-config \
+        sudo \
+	tmux \
+	xz-utils \
+	ufw
```

Comment:
As a debian user and ubuntu hater I really don't like ufw. As the script is not using it as well, probably we should leave that to users to decide how to do port forwarding.